packages/execution-environments/src/common/endowments/network.test.ts

```diff
@@ -514,4 +534,85 @@ describe('Network endowments', () => {
       await allClosed;
     });
   });
+
+  describe('ResponseWrapper', () => {
+    beforeEach(() => {
+      fetchMock.enableMocks();
+    });
+
+    afterEach(() => {
+      fetchMock.disableMocks();
+    });
+
+    it('should return for each of the Response methods', async () => {
+      const RESULT = 'OK';
+      fetchMock.mockOnce(async () => RESULT);
+
+      const { fetch } = network.factory();
+      const result = await fetch('foo.com');
+
+      expect(result).toBeDefined();
+      expect(await result.text()).toBe(RESULT);
```

Comment:
you can add `expect(result.bodyUsed).toBe(false)` above

Reply:
Added 👍